src/main/kotlin/org/jetbrains/bio/big/Bed.kt

```diff
@@ -211,12 +215,9 @@ data class BedEntry(
         val chunks = IntArray(size)
         val s = Splitter.on(',').split(this).iterator()
         var ptr = 0
-        // actual fields my be less that size, but not vice versa
-        check(s.hasNext() == (size > 0))
-        while (s.hasNext() && ptr < size) {
+        while (ptr < size) {
             chunks[ptr++] = s.next().toInt()
```

Comment:
Not clear what for is `More strict splitToInts`. Also if `s` has fewer chunks then `size` this code will throw an AOBE?

I don't remember the exact use case, probably is when a user needs rgb-color and only one array based column, e.g. (block sizes or block starts), so uses '0' or '.' for the unused column.

Reply:
Standard conformance, mostly:
> blockSizes - A comma-separated list of the block sizes. The number of items in this list should correspond to blockCount.

We can revert this change, I don't have any particularly strong feelings for it. Just say the word.
Yes, `IOOBE` will be thrown. It will also be immediately caught at both call sites of this private function.
`color` is column 9, while `blockStarts` and `blockSizes` are 12 and 11 respectively, so the use case must be different.

Reply:
Ok, let's leave as is until we've faced some real issue